continuous_signals/operations.ipynb

```diff
@@ -41,7 +41,7 @@
    "source": [
     "**Example**\n",
     "\n",
-    "The following example illustrates the superposition of two harmonic signals $x(t) = A \\cdot \\cos(\\omega_1 t) + B \\cdot \\cos(\\omega_2 t)$ with weights $A$, $B$ and angular frequencies $\\omega_1$ and $\\omega_2$."
+    "The following example illustrates the superposition of two harmonic signals $x_\\text{A}(t) = A \\cdot \\cos(\\omega_1 t)$ and  $x_\\text{B}(t) = B \\cdot \\cos(\\omega_2 t)$ with weights $A$, $B$ and angular frequencies $\\omega_1$ and $\\omega_2$."
```

Comment:
Should we use \omega_\text{A} instead of \omega_1 also here?

Reply:
yes, makes it even more consistent. I've changed in b0ab931a257ae90a1b7c11ec131941ae3cf70ba2